idunn/geocoder/nlu_client.py

```diff
@@ -237,6 +242,22 @@ async def get_intentions(self, text, lang, focus=None):
                     if brand_query or intention.filter.category:
                         intentions.append(intention)
 
+        logs_extra = {
+            "lang": lang,
+            "focus": focus,
+            "tags_list": tags_list,
+            "intentions": [x.dict() for x in intentions],
+            "brand_query": brand_query,
+            "cat_query": cat_query,
+            "place_query": place_query,
+            "skip_classifier": skip_classifier,
+        }
```

Comment:
Same as above.

Moreover, I am not sure about using the raw `intentions` list in the log message. It could make the mapping of the data storage for these logs hard to maintain.

Reply:
I just removed it then, other fields are probably light enough (and generally most of them are empty)